packages/react-components/src/localization/locales/fr-FR/strings.json

```diff
@@ -61,7 +64,17 @@
     "friday": "Vendredi",
     "saturday": "Samedi",
     "participantJoined": "a rejoint la conversation.",
-    "participantLeft": "a quitté le conversation."
+    "participantLeft": "a quitté le conversation.",

```

Comment:
Should be "a quitté **la** conversation."

Reply:
yep, missed this one :)